### ISSUE TITLE
Update to yellowstone-grpc 2.0.0 intoduces zeroize version conflict

### ISSUE THREAD

**Issue by 808putnam** [rpcpool/yellowstone-vixen#31]
Attempting to cargo build with the latest cargo.toml changes setting yellowstone-grpc to version 2.0.0 produces the zeroize version conflict error as seen below. Reverting back to versions 1.16.0 and 1.15.0 - I am able to build successfully.

Updating crates.io index
    Updating git repository `https://github.com/rpcpool/yellowstone-grpc`
error: failed to select a version for `zeroize`.
    ... required by package `rustls v0.23.0`
    ... which satisfies dependency `rustls = "^0.23"` of package `tokio-rustls v0.26.0`
    ... which satisfies dependency `tokio-rustls = "^0.26"` of package `tonic v0.12.1`
    ... which satisfies dependency `tonic = "^0.12.1"` (locked to 0.12.1) of package `yellowstone-grpc-client v2.0.0 (https://github.com/rpcpool/yellowstone-grpc?branch=master#50963951)`
    ... which satisfies git dependency `yellowstone-grpc-client` of package `yellowstone-vixen v0.0.0 (/workspaces/qtrade/yellowstone-vixen/crates/runtime)`
    ... which satisfies path dependency `yellowstone-vixen` (locked to 0.0.0) of package `example-vixen-prometheus v0.0.0 (/workspaces/qtrade/yellowstone-vixen/examples/prometheus)`
versions that meet the requirements `^1.7` are: 1.8.1, 1.7.0

all possible versions conflict with previously selected packages.

  previously selected package `zeroize v1.3.0`
    ... which satisfies dependency `zeroize = "^1.3"` (locked to 1.3.0) of package `solana-zk-token-sdk v2.0.9`
    ... which satisfies dependency `solana-zk-token-sdk = "^2.0.9"` (locked to 2.0.9) of package `yellowstone-vixen-parser v0.0.0 (/workspaces/qtrade/yellowstone-vixen/crates/parser)`
    ... which satisfies path dependency `yellowstone-vixen-parser` (locked to 0.0.0) of package `example-vixen-prometheus v0.0.0 (/workspaces/qtrade/yellowstone-vixen/examples/prometheus)`

failed to select a version for `zeroize` which could resolve this conflict

**Comment by notwedtm** [comment]
@808putnam Are you willing to share details on how you resolved this?

**Comment by 808putnam** [comment]
Below is how I modified the cargo.toml to get it to build. Still need to take it one step further and attempt to get an example running:

[workspace]
members = ["crates/*", "examples/*"]
resolver = "2"

[workspace.dependencies]
yellowstone-grpc-client = { version = "1.16.0", git = "https://github.com/rpcpool/yellowstone-grpc", branch = "master" }
yellowstone-grpc-proto = { version = "1.15.0", git = "https://github.com/rpcpool/yellowstone-grpc", branch = "master", default-features = false }
yellowstone-vixen = { path = "crates/runtime", version = "0.0.0" }
yellowstone-vixen-core = { path = "crates/core", version = "0.0.0" }
yellowstone-vixen-mock = { path = "crates/mock", version = "0.0.0" }
yellowstone-vixen-parser = { path = "crates/parser", version = "0.0.0" }
yellowstone-vixen-proto = { path = "crates/proto", version = "0.0.0" }

**Comment by Taylor123** [comment]
I've been able to patch in my project with the following:

```
[patch.crates-io.curve25519-dalek]
git = "https://github.com/solana-labs/curve25519-dalek.git"
rev = "b500cdc2a920cd5bff9e2dd974d7b97349d61464"
```

There are notes of the zeroize issues in the [legacy Solana repo](https://github.com/solana-labs/solana/blob/master/Cargo.toml#L514). Seems that there is now a forked version of the curves create in the Agave repo, which I would presume also helps fix the underlying issue.

**Comment by notwedtm** [comment]
That did it! Thank you!

**Comment by Nagaprasadvr** [comment]
fixed in #34

**Comment by 808putnam** [comment]
Thanks guys! And here are some inputs from Agave regarding fixing this in v2.1:
https://github.com/anza-xyz/agave/issues/3337

Now - time to do some parsing!
